db/seeds.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
+require 'factory_bot_rails'
+
 GENERATE_TAGS_COUNT = 30
 
 FactoryBot.create_list(:user, 6)
 FactoryBot.create_list(:tag, GENERATE_TAGS_COUNT)
+GENERATE_POSTS_COUNT = 100
+
+FactoryBot.create_list(:user, 6)
+FactoryBot.create_list(:post, GENERATE_POSTS_COUNT)
```

Comment:
If understand correctly, we don't have any post with likes, correct?

Reply:
agreed, @skripchenkoVeta please add post factory trait :with_likes or smth like that and add some posts with likes to seeds